docker_boot.sh

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
-./wait-for-it.sh database2:3306 -- php artisan migrate
+set -e
 
-php-fpm
+DB_HOST="${DB_HOST:?environment variable is missing}"
+DB_PORT="${DB_PORT:-3306}"
+
+./wait-for-it.sh "${DB_HOST}:${DB_PORT}" -- php artisan migrate --force
+
+# php artisan optimize
```

Comment:
What is this supposed to do?

Reply:
The optimize command precompiles the config, routes, services and providers so they don't need to be parsed for every request. For example in config files there are a few calls to the env(), but these do not change after starting the container so the resolved values can be cached. Just a little optimization.

But when I tried to run it, it failed because there was a route with an anonymous function, and those can't be cached. So I commented out the call.

Reply:
Ah, right. I recently read on Taylor's Twitter that closures will be cacheable in a future version of Laravel, but that doesn't do shit for us right now 😂